CloudKit-demo/Classes/Managers/CloudKitManager.swift

```diff
@@ -95,4 +94,19 @@ final class YALCloudKitManager {
         })
     }
     
+    //MARK: check that user is logged
+    static func checkLoginStatus(handler: () -> Void) {
```

Comment:
@DarthRumata this method calls a completion handler just for one case (when account is available). How do you feel if app will call completion handler for all cases ? I suggest to pass a bool value in the completion handler and check it in the calling code. Thanks.